sql/pgwire_test.go

```diff
@@ -443,8 +444,10 @@ func TestPGPreparedQuery(t *testing.T) {
 
 	runTests := func(query string, tests []preparedQueryTest, queryFunc func(...interface{}) (*sql.Rows, error)) {
 		for _, test := range tests {
-			rows, err := queryFunc(test.params...)
-			if err != nil {
+			if testing.Verbose() || log.V(1) {
+				fmt.Printf("query: %s\n", query)
```

Comment:
does this actually help? we already log the query in the error messages. if it helps to log the verb ("query"), why not add it there? (it shouldn't help, since you get a line number already, which will point to this function).